doc/getting-started/building-from-source.rst

```diff
@@ -252,29 +252,18 @@ Now you can build all these components at once as follows:
 
 The build results are found in the ``build`` directory. You can specify
 a different Output folder by setting the ``O`` ``make`` parameter,
-for example: ``make O=build-nuc BOARD=nuc7i7dnb``.
+for example: ``make O=build-nuc``.
 
-If you only need the hypervisor, use this command:
 
 .. code-block:: none
 
-   $ make clean                       # Remove files previously built
-   $ make -C hypervisor
-   $ make -C misc/efi-stub HV_OBJDIR=$PWD/hypervisor/build EFI_OBJDIR=$PWD/hypervisor/build
-
-The ``acrn.efi`` will be generated in the ``./hypervisor/build/acrn.efi`` directory hypervisor.
-
-As mentioned in :ref:`ACRN Configuration Tool <vm_config_workflow>`, the
-Board configuration and VM configuration can be imported from XML files.
-If you want to build the hypervisor with XML configuration files,
-specify the file location as follows (assuming you're at the top level
-of the acrn-hypervisor directory):
-
-.. code-block:: none
-
-   $ make BOARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
+   $ make all OARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
    SCENARIO_FILE=$PWD/misc/vm_configs/xmls/config-xmls/nuc7i7dnb/industry.xml TARGET_DIR=xxx
 
+The build results are found in the ``build`` directory. You can specify
+a different Output folder by setting the ``O`` ``make`` parameter,
+for example: ``make O=build-nuc``.
```

Comment:
Can you also briefly explain what the `TARGET_DIR` parameter does?

Reply:
@gvancuts  The  TARGET_DIR parameter is explained in Note:
The TARGET_DIR parameter specifies what directory is used to store configuration files imported from XML files. If the TARGET_DIR is not specified, the original configuration files of acrn-hypervisor would be overridden.